apps/apple/package.json

```diff
@@ -26,5 +26,8 @@
   },
   "jest": {
     "preset": "react-native"
+  },
+  "workspaces": {
+    "nohoist": ["react-native", "react-native/**", "@babel/**"]
```

Comment:
Not part of the review, but is this something we should be doing for all the test app platforms?

Reply:
Win32/Windows may not need to- I haven't checked but they have a completely separate fork of react-native which may have different requirements. iOS/macOS need it though

Reply:
Can we understand this before going this step? I'm pretty sure other platforms have a dependency on babel. If it's a problem in our fork of react-native we should be sure upgrading will fix it or otherwise just fix it. but adding this allowance for it here sounds non-preferred from what I can read of yarn

Reply:
And this feels like the sort of thing where if we say we'll do it as a follow-up it may just never happen. So probably at the very least need a comment explaining this and how a future contributor should treat this line (add no more, look to remove, why it wasn't removed to begin with)

Reply:
Windows does too but it their version of RN is very different from ios/android/macos. RN expects babel to be at a specific location but yarn automatically pulls it up to root. That's why we need to nohoist it to get it back to where react native is expecting that dependency to be. This is a common problem using yarn in monorepos and nohoist is the tool designed exactly for it

Reply:
If windows has this fixed in their version of react-native, does that mean we can too?

Reply:
I can try being more specific. I think we only need the runtime component from within babel

Reply:
Comments aren't allowed in json

Reply:
Maybe... I don't think it's a big enough issue to merit spending much time on right now when nohoist is designed for this exact problem and recommended to use in react-native monorepos

Reply:
But shouldn't autolinking in react-native have removed this need? The worry is that there's still something deeply wrong with our react-native fork and we'll have to keep working around it

Reply:
It's likely not a deep problem with our fork of react-native. We don't do anything unusual with babel so it's more likely an artifact of trying to consume it in a monorepo with tons of other packages